include/nbl/asset/utils/ICPUVirtualTexture.h

```diff
@@ -133,6 +133,7 @@ class ICPUVirtualTexture final : public IVirtualTexture<ICPUImageView, ICPUSampl
         upscaled_img->setBufferAndRegions(std::move(buf), std::move(regions));
 
         using blit_filter_t = asset::CBlitImageFilter<asset::VoidSwizzle,asset::IdentityDither/*TODO: White Noise*/,void,false,asset::CMitchellImageFilterKernel<>>;
+        using blit_utils_t = asset::CBlitUtilities<asset::CMitchellImageFilterKernel<>>;
```

Comment:
it would be wise to add a `using` public alias inside the `CBlitImageFilter` class so you could do `blit_filter_t::utils_t`

Reply:
It was already there, now renamed it to `blit_utils_t`, for clarity.

Resolved.